client/packages/electron/src/electron.ts

```diff
@@ -167,7 +171,25 @@ const start = (): void => {
       hardwareId,
     });
   });
+
+  window.webContents.on(
+    'did-fail-load',
+    (_event, _errorCode, errorDescription, validatedURL) => {
+      // not strictly necessary, done to prevent an infinite loop if the loadFile fails
+      if (hasLoadingError) return;
+
+      hasLoadingError = true;
+      const search = `error=Failed to load URL ${validatedURL} with error: ${errorDescription}`;
+      // loading './error.html' should work but doesn't, either as loadURL or loadFile
```

Comment:
it really should've worked to loadURL('./error.html') but it doesn't. that fails quietly. 
Trying to loadFile fails with an error.